datadog/resource_datadog_dashboard.go

```diff
@@ -1174,6 +1188,22 @@ func getGroupDefinitionSchema() map[string]*schema.Schema {
 			Optional:    true,
 			Description: "The title of the group.",
 		},
+		"background_color": {
```

Comment:
The PR description didn't mention this field; is this intended to be on this PR?

Reply:
Yes it is, I just forgot it. It was not added in DataDog/datadog-api-spec#944 but already before here: https://github.com/DataDog/datadog-api-spec/pull/912